include/natalie/integer_object.hpp

```diff
@@ -13,6 +13,7 @@
 #include "natalie/object.hpp"
 #include "natalie/rounding_mode.hpp"
 #include "natalie/types.hpp"
+#include "nathelpers/typeinfo.hpp"
```

Comment:
I could not get the gc_lint file to skip the structs in this file in a non-hacky way, so I resorted to moving this file into a different directory.